filterdb/db.go

```diff
@@ -51,6 +51,9 @@ type FilterDatabase interface {
 	// target block hash cannot be found, then ErrFilterNotFound is to be
 	// returned.
 	FetchFilter(*chainhash.Hash, FilterType) (*gcs.Filter, error)
+
+	// PurgeFilters purge all filters with a given type from persistent storage
+	PurgeFilters(FilterType) error
```

Comment:
dead code?

Reply:
Not used within neutrino but intended to be used by apps that use neutrino as a chain service.
For example, we use a background job that run neutrino  to persist the filters and this gives us a way to purge the filters when not needed.